tests/dummy/app/controllers/nodes/detail/draft-registrations/detail.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    editedMetadata: {},
```

Comment:
For future reference, ember controllers are singletons, which means that a single instance might be shared between visits- that makes shared storage like this a potential source of subtle bugs.

When we do make UI improvements, we'll want to be careful to ensure that a user navigating within the app won't see state left over from a previous visit to the page. Not a blocker for this PR though.